tools/src/h5diff/ph5diff_main.c

```diff
@@ -173,7 +173,7 @@ ph5diff_worker(int nID)
 
                 /* When get token, send all of our output to the manager task and then return the token */
                 for (i = 0; i < outBuffOffset; i += PRINT_DATA_MAX_SIZE)
-                    MPI_Send(outBuff + i, PRINT_DATA_MAX_SIZE, MPI_BYTE, 0, MPI_TAG_PRINT_DATA,
+                    MPI_Send(outBuff + i, PRINT_DATA_MAX_SIZE, MPI_CHAR, 0, MPI_TAG_PRINT_DATA,
```

Comment:
The manager rank specifies `MPI_CHAR` when receiving these messages. It doesn't seem to make much of a difference anyway for char vs. byte, but it seems better to make sure we match the types anyway.